frontend/javascripts/oxalis/model/sagas/volumetracing_saga.js

```diff
@@ -110,13 +110,17 @@ export function* editVolumeLayerAsync(): Generator<any, any, any> {
       );
     }
 
+    let lastPosition = startEditingAction.position;
     while (true) {
       const { addToLayerAction, finishEditingAction } = yield* race({
         addToLayerAction: _take("ADD_TO_LAYER"),
         finishEditingAction: _take("FINISH_EDITING"),
       });
 
-      if (finishEditingAction) break;
+      if (finishEditingAction) {
+        lastPosition = null;
```

Comment:
Does `lastPosition` actually need to be set to `null` here? I would expect not, because we break out of the loop anyways.